custom_components/myskoda/translations/en.json

```diff
@@ -54,20 +54,20 @@
                     "on": "Open"
                 }
             },
-            "lights_on": {
-                "name": "Lights",
-                "state": {
-                    "off": "Off",
-                    "on": "On"
-                }
-            },
             "locked": {
                 "name": "Vehicle Locked",
                 "state": {
                     "off": "Locked",
                     "on": "Unlocked"
                 }
             },
+            "parkinglights_on": {
```

Comment:
Other translations should be updated as well?

Reply:
Yes, once it is pushed to main, it will be added as a translatable string in Crowdin, where people can translate it without having to know git :grin: